Docs/README_AlternateBuildManifests.md

```diff
@@ -0,0 +1,36 @@
+# Alternate Build-Manifest Capability
+
+As of this document's author-date (2020-03-06), the AMIgen7 scripts now support the creation of AMIs with customized build-manifests. Two modes are supported:
+
+- **Groups:** Packages included in the AMI are as maintained in the AMI's source yum repository
```

Comment:
This sentence is a bit confusing. It would be more clear to link the group to to the yum repository in the text. Something like:
```suggestion
- **Groups:** Packages included in the AMI will be sourced from the corresponding group in the AMI's source yum repository
```